revert removed directories

garaemon opened the pull request:
The pull request by k-okada is already merged into hydro-devel, but not merged into groovy-devel
https://github.com/ros-perception/perception_pcl/pull/11

This pull request not only includes the diff of the k-okada pull request but also reverts `pcl_nodelets.xml`.

`pcl_nodelets.xml` is required to be reverted in order to use the nodelet classes from other ros launch files.

k-okada committed:
copy features/segmentation/surface from fuerte-devel

k-okada committed:
fix to compileable

garaemon committed:
Merge branch 'groovy-devel' of https://github.com/garaemon/perception_pcl into groovy-devel

garaemon committed:
reverting the pcl_nodelets.xml to use pcl_feature and so on

wjwwood commented:
You will need to add a `.travis.yml` file to this pull request before the build will pass. I recommend copying the one from the hydro branch and replacing hydro with groovy:

https://github.com/ros-perception/perception_pcl/blob/hydro-devel/.travis.yml

garaemon committed:
using pcl::PointNormal instead of pcl::Normal in MovingLeastSquares nodelet

garaemon committed:
adding .travis.yml (copied from hydro-devel)

garaemon committed:
running travis-ci

garaemon committed:
using groovy in runnning on travis

garaemon committed:
fixing the path to be sourced on travis

garaemon committed:
adding dependency of generating cfg to the library targets

garaemon committed:
fixing CMakeLists.txt to be able to be compiled on travis, adding dependency to gencfg

garaemon committed:
using dynamic_reconfigure.parameter_generator_catkin instead of dynamic_reconfigure.parameter_generator for the cfg files

garaemon commented:
added .travis.yml.
fixed CMakeLists.txt to pass the travis-ci.
fixed moving_least_squares.h because it used `pcl::Normal` which is not supported by `pcl::MovingLeastSquares`.

https://travis-ci.org/ros-perception/perception_pcl/builds/13753913
